tests/storage/storage.go

```diff
@@ -1314,24 +1277,23 @@ var _ = SIGDescribe("Storage", func() {
 						},
 					},
 				}
+
+				vmi1 := libvmi.NewAlpine(libvmi.WithPersistentVolumeClaim(diskName, pvcClaim, v1.DiskBusVirtio))
+				vmi2 := libvmi.NewAlpine(libvmi.WithPersistentVolumeClaim(diskName, pvcClaim, v1.DiskBusVirtio))
+
 				vmi1.Labels = labels
 				vmi2.Labels = labels
 
 				vmi1.Spec.Affinity = affinityRule
 				vmi2.Spec.Affinity = affinityRule
 
-				diskName := "disk1"
-				pvcClaim := "pvc-test-disk1"
 				libstorage.CreateBlockPVC(pvcClaim, "500Mi")
-				tests.AddPVCDisk(vmi1, diskName, v1.DiskBusVirtio, pvcClaim)
-				tests.AddPVCDisk(vmi2, diskName, v1.DiskBusVirtio, pvcClaim)
-
 				setShareable(vmi1, diskName)
 				setShareable(vmi2, diskName)
 
 				By("Starting the VirtualMachineInstances")
-				tests.RunVMIAndExpectLaunch(vmi1, 500)
-				tests.RunVMIAndExpectLaunch(vmi2, 500)
+				vmi1 = tests.RunVMIAndExpectLaunch(vmi1, 500)
+				vmi2 = tests.RunVMIAndExpectLaunch(vmi2, 500)
```

Comment:
Not in this PR, but note that the test time can be improved by starting both in parallel and waiting for both serially.
i.e.:
```
create(vmi1)
create(vmi2)
wait(vmi1)
wait(vmi2)
```